app/src/main/java/com/sunlightlabs/android/congress/notifications/subscribers/RollsLegislatorSubscriber.java

```diff
@@ -23,10 +23,9 @@ public String decodeId(Object result) {
 	@Override
 	public List<?> fetchUpdates(Subscription subscription) {
 		Utils.setupAPI(context);
-		String chamber = subscription.data;
```

Comment:
We're still storing the chamber field on member vote subscriptions, just in case it's needed again in the future, but it's not needed to filter the API as far as I can tell and does not need to be factored into the checking/fetching pipeline.